docs/tutorials/scheduling.rst

```diff
@@ -353,7 +356,236 @@ and run the priority scheduler again.
     # To get a plot of the airmass vs where the blocks were scheduled
     plt.figure(figsize = (14,6))
     plot_schedule_airmass(priority_schedule)
+    plt.tight_layout()
     plt.legend(loc="upper right")
     plt.show()
 
-Nothing new shows up because Alpha Centauri isn't visible from APO.
\ No newline at end of file
+Nothing new shows up because Alpha Centauri isn't visible from APO.
+
+.. _scheduling-user_defined_schedulers:
+
+User-Defined Schedulers
+=======================
+
+There are many ways that targets can be scheduled with, only two of which
+are currently implemented. This example will walk through the steps for
+creating your own scheduler that will be compatible with the tools of
+the ``scheduling`` module.
+
+As you may have noticed above, the schedulers are assembled by making a
+call to the initializer of the class (e.g. `~astroplan.scheduling.PriorityScheduler`).
+Each of the schedulers is subclassed from the abstract `astroplan.scheduling.Scheduler`
+class, and our custom scheduler needs to be as well.
+
+For our scheduler, we will make one starts at the beginning and schedules
+the first ``ObservingBlock`` that it finds that doesn't have a score of zero: a
```

Comment:
I added some explanation of score/constraints, but the example scheduler doesn't actually try to maximize score. In a future commit I have added a few comments that talk about how a different scheduler might try to maximize the score and how scores are evaluated.